docs/components/board/customlinux.md

```diff
@@ -213,15 +230,104 @@ On your `customlinux` board, create a file in your <file>/home/root</file> direc
 {{% /tab %}}
 {{< /tabs >}}
 
-The following parameters are available for pin mappings:
+The following parameters are available for each pin object:
 
 <!-- prettier-ignore -->
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `name` | string | **Required** | The name of the pin. <br> Example: `"3"`. |
-| `device_name` | string | **Required** | The device name. <br> Example: `"gpiochip0"`. |
-| `line_number` | integer | **Required** | The line number of the pin. <br> Example: `81`. |
-| `pwm_id` | integer | **Required** | The power management id of the pin. Example: `-1`. |
+| `name` | string | **Required** | The name of the pin. This can be anything you want but it is convenient to use the physical board pin number. <br> Example: `"3"`. |
+| `device_name` | string | **Required** | The name of the device in <file>/dev</file> that this pin is attached to. Multiple pins may be attached to the same GPIO chip.  See [GPIO info tips](#tips-for-finding-gpio-information) below. <br> Example: `"gpiochip0"`.
+| `line_number` | integer | **Required** | The line on the chip that is attached to this pin. See [GPIO info tips](#tips-for-finding-gpio-information) below. <br> Example: `81`. |
+| `pwm_chip_sysfs_dir` | string | Optional | Uniquely specifies which PWM device within [sysfs](https://en.wikipedia.org/wiki/Sysfs) this pin is connected to. See [PWM info tips](#tips-for-finding-pwm-information) below. <br> Example: `3290000.pwm`. |
+| `pwm_id` | integer | Optional | The power management ID of the pin. See [PWM info tips](#tips-for-finding-pwm-information) below. <br> Example: `-1`. |
```

Comment:
Not for power management. It's the line number on the PWM (pulse width modulated) device, similar to the line number on the GPIO device from 2 lines earlier.

Reply:
```suggestion
| `pwm_id` | integer | Optional | The line number on the PWM chip. See [PWM info tips](#tips-for-finding-pwm-information) below. <br> Example: `-1`. |
```

Reply:
I got hung up on the "power management" part, and didn't notice that the example is negative. We've been using -1 for nonexistent PWM lines, but for all the real ones, it'll be non-negative. Make the example 0 or 1.